.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: node_js
 node_js:
   - "lts/carbon"
-  - "lts/dubnium"
```

Comment:
We should still build on the latest LTS, but just allow it to fail for now. We don't want to lose sight of this - we still need to resolve the issue ASAP.